src/execute-pr-actions.ts

```diff
@@ -180,38 +178,19 @@ function makeComment(body: string, tag: string) {
 }
 
 async function getProjectBoardColumnIdByName(name: string): Promise<string> {
-    const data = await cache.getAsync("project board colum names", Infinity, async () => {
-        const res = await query<GetProjectColumnsResult>(GetProjectColumns);
-        return res.repository?.project?.columns.nodes ?? [];
-    });
-    const res = data.filter(e => e && e.name === name)[0]?.id;
+    const columns = await getProjectBoardColumns();
+    const res = columns.filter(e => e && e.name === name)[0]?.id;
     if (res !== undefined) {
         return res;
     }
     throw new Error(`No project board column named "${name}" exists`);
 }
 
-async function getLabelIdByName(name: string): Promise<string> {
-    const data = await cache.getAsync("label ids", Infinity, async () => {
-        const res = await query<GetLabelsResult>(GetLabels);
```

Comment:
These queries moved to `cachedQueries.ts` so they can be easily mocked in tests.